### DISCUSSION TITLE
채팅 데이터를 NoSql에 저장하는 것은 어떨까요?

### DISCUSSION THREAD

**Discussion by Starlight258** [Attica-org/athens-backend#19] [Ideas]
@big-cir 분이 이야기했던 것처럼 채팅에 굳이 RDB를 쓸 필요가 없을 것 같다.
굳이 무결성이 필요할까? 그리고 채팅은 대용량 데이터라 NoSql을 사용하면 이점이 있을 것 같다.

실무에서도 RDB에 NoSql을 함께 쓴다고 하니 고려해보자.

@yybmion @big-cir

**Comment by yybmion** [comment]
관련된 주제로 예전에 제가 흥미롭게 봤던 노마드 코더님의 '디스코드가 10조 메시지를 저장하는 방법?!' 의 유튜브 영상 올려놓겠습니다

여유로울 때 보면 재밌을 듯 합니다.
@Starlight258 @big-cir 

https://www.youtube.com/watch?v=oIaQuX7Nk7c

**Reply by Starlight258** [reply]
굉장히 흥미로운 주제네요! 영상 공유 감사드려요 :)
정기회의에서 저희 서비스에 적용할 부분이 있을지 의견 나눠봐도 좋을 것 같아요~
